torch/csrc/jit/symbolic_script.cpp

```diff
@@ -439,6 +450,15 @@ const std::vector<std::string> functions = {
 
             return torch.full_like(self, fill_value), backward
 
+        def lerp(self,
+                 end,
+                 weight: float):
+            def backward(grad_output):
```

Comment:
can we support tensor lerp too?

Reply:
sure, but i did not see the tensor lerp in the documentation (although it's in schema..)

Reply:
Sorry for not being clear, I meant the case where `weight` is a tensor.  That seems to be already in doc, although it is indeed to miss that.... https://pytorch.org/docs/master/torch.html#torch.lerp

Reply:
gotcha, yeah that's also what I mean lol. it looks like [stable doc](https://pytorch.org/docs/stable/torch.html#torch.lerp) don't have weight as tensor but master doc have it, I guess it's recently added.

Reply:
yes it is added after 1.0.1. thanks :)

Reply:
just added it :)